lib/filter/tests/test_filters_in_list.c

```diff
@@ -43,6 +43,7 @@
 #define LIST_FILE_DIR "%s/lib/filter/tests/filters-in-list/"
 
 static MsgFormatOptions parse_options;
+char *top_srcdir = NULL;
```

Comment:
Wouldn't be `char *top_srcdir = TOP_SRCDIR;` sufficient here? Do we need strdup/free in setup/teardown?

Reply:
Hi @furiel ,

Yes, the static allocation is enough. I will change the code.

Reply:
Thanks!